src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentTypeFinder.cs

```diff
@@ -133,152 +154,11 @@ internal static partial class DependentTypeFinder
             return result.SelectAsArray(t => (t.GetSymbolKey(), solution.GetOriginatingProjectId(t)));
         }
 
-        /// <summary>
-        /// Used for implementing the Inherited-By relation for progression.
-        /// </summary>
-        public static Task<ImmutableArray<INamedTypeSymbol>> FindImmediatelyDerivedClassesAsync(
-            INamedTypeSymbol type,
-            Solution solution,
-            CancellationToken cancellationToken)
-        {
-            return FindTypesFromCacheOrComputeAsync(
-                type, solution, projects: null,
-                cache: s_typeToImmediatelyDerivedClassesMap,
-                findAsync: c => FindDerivedClassesAsync(type, solution, projects: null,
-                    transitive: false, cancellationToken: c),
-                cancellationToken: cancellationToken);
-        }
```

Comment:
to help with clutter (and esp. because OOP adds another layer), the individual ops moved to their own files.  They did change slightly, but i call out how in the PR later.